### DISCUSSION TITLE
pocketbase js

### DISCUSSION THREAD

**Discussion by SrimanNarayanaYendluri** [pocketbase/pocketbase#4792] [Q&A]
Can i crate collection through java script how?

**Comment by ganigeorgiev** [comment]
If you want to create collections via the Web APIs you could check https://pocketbase.io/docs/api-collections/.

If you want to create collections via the JSVM, then you could check:
- https://pocketbase.io/docs/js-collections/
- https://pocketbase.io/docs/js-migrations/

If you have further questions, please don't just "prompt" and try to elaborate a little more on your use case to better understand what you are trying to do.

**Reply by SrimanNarayanaYendluri** [reply]
ok thank you
